SDKIntegrationTestApps/PostRelease-iOSReleaseTest-Carthage/iOSReleaseTestTests/iOSReleaseTestTests.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  iOSReleaseTestTests.swift
+//  iOSReleaseTestTests
+//
+//  Created by Nidhi Dixit on 1/15/23.
+//
+
+import XCTest
+import BranchSDK
+
+final class iOSReleaseTestTests: XCTestCase {
+
+    override func setUpWithError() throws {
+        // Put setup code here. This method is called before the invocation of each test method in the class.
+    }
+
+    override func tearDownWithError() throws {
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
+    }
+
+    func testSetTrackingDisabled() throws {
```

Comment:
I'd like to expand the tests and integration we do. But this is a good starting point.